packages/ember/tests/reexports_test.js

```diff
@@ -25,9 +25,7 @@ moduleFor(
           assert,
           path,
           moduleId,
-          exportName,
-          isDeprecated,
-          `Ember.${path} exports correctly`
```

Comment:
This 7th argument (a string description) is not used. Here it is removed.

Note: also see https://github.com/emberjs/ember.js/pull/19653/files#r671884855 regarding the removal of `isDeprecated` on the line above.